config/boot.rb

```
@@ -1,5 +1,4 @@
-# frozen_string_literal: true
-
 ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)
```

RuboCop (1.85.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)